yeat/tests/data/configs/canu.cfg

```diff
@@ -0,0 +1,17 @@
+{
+    "samples": {
+        "sample1": [
+            "yeat/tests/data/ecoli.fastq.gz"
+        ]
+    },
+    "assemblers": [
+        {
+            "label": "canu-default",
+            "algorithm": "canu",
+            "extra_args": "genomeSize=4.8m",
+            "samples": [
+                "sample1"
+            ]
+        }
+    ]
+}
```

Comment:
Created a lot of config test files. As a result, to organize them, I created a dedicated `configs` directory in `yeat/tests/data`.